catbuffer/parser/catparser/ast.py

```diff
@@ -339,6 +388,36 @@ def __init__(self, tokens, disposition=None):
 
 		self.attributes = None
 
+	@property
+	def is_const(self):
+		"""Returns true if this field is const."""
+		return 'const' == self.disposition
+
+	@property
+	def is_reserved(self):
+		"""Returns true if this field is reserved."""
+		return 'reserved' == self.disposition
+
+	@property
+	def is_size_reference(self):
+		"""Returns true if this field is a size reference."""
+		return 'sizeof' == self.disposition
+
+	@property
+	def is_conditional(self):
+		"""Returns true if this field is conditional."""
+		return isinstance(self.value, Conditional)
+
+	@property
+	def display_type(self):
+		"""Gets the display type."""
+		return DisplayType.UNSET if isinstance(self.field_type, str) else self.field_type.display_type
+
+	@property
+	def size(self):
+		"""Gets the backing size."""
+		return self.field_type.size if hasattr(self.field_type, 'size') else None
+
```

Comment:
this whole piece with properties looks just great (I mean the fact that it looks like cosmetic compared to old)